Detections/Box/BoxMultipleItemsDeletedByUser.yaml

```diff
@@ -0,0 +1,27 @@
+id: 1b212329-6f2c-46ca-9071-de3464f3d88d
+name: Box - Many items deleted by user
+description: |
+  'Detects when a user deletes many items in short period of time.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 5m

```

Comment:
5m is not needed here.  You are looking at the last 1h every 5m.  We don't recommend 5m, but if it is needed, then queryPeriod should also be 5m if the goal is greater than 100 deleted items in 5m.  If it is, then set to 1h frequency and use bin(TimeGenerated, 5m) which will give you the count every 5m to check the threshold match but only run hourly.

Reply:
updated